test/test_host.go

```diff
@@ -679,3 +681,94 @@ func (s *HostSuite) TestUpdateTags(t *c.C) {
 		t.Fatal("expected tag to be deleted but is still present")
 	}
 }
+
+func (s *HostSuite) TestLogSinks(t *c.C) {
+	// deploy custom logaggregator app
+	client := s.controllerClient(t)
+	logApp := &ct.App{Name: "test-logaggregator"}
+	t.Assert(client.CreateApp(logApp), c.IsNil)
+	release, err := client.GetAppRelease("logaggregator")
+	t.Assert(err, c.IsNil)
+	proc := release.Processes["app"]
+	proc.Env = map[string]string{"SERVICE_NAME": logApp.Name}
+	syslogPort := 55514
+	proc.Ports[1].Port = syslogPort
+	release.Processes["app"] = proc
+	release.ID = ""
+	t.Assert(client.CreateRelease(release), c.IsNil)
+	t.Assert(client.SetAppRelease(logApp.ID, release.ID), c.IsNil)
+	t.Assert(client.PutFormation(&ct.Formation{
+		AppID:     logApp.ID,
+		ReleaseID: release.ID,
+		Processes: map[string]int{"app": 1},
+	}), c.IsNil)
+	defer client.DeleteApp(logApp.ID)
+
+	// add sink to controller
+	config, err := json.Marshal(&ct.SyslogSinkConfig{
+		URL: fmt.Sprintf("tcp://%s.discoverd:%d", logApp.Name, syslogPort),
+	})
```

Comment:
Should this test the prefix template? Also, the scheme is `tcp://` but I think it should be `syslog://`.